moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -162,6 +162,10 @@ schemas["testpilottest"]                        = schemas.vacuous
 schemas["malware-addon-states"]                 = schemas.vacuous
 schemas["sync"]                                 = schemas.vacuous
 schemas["outofdate-notifications-system-addon"] = schemas.vacuous
+schemas["x-shield-studies"]                     = schemas.vacuous
+schemas["shield-study"]                         = schemas.vacuous
+schemas["x-contextual-feature-recommendation"]  = schemas.vacuous
+schemas["FirefoxOS"]                            = schemas.vacuous
```

Comment:
sigh... just detected another valid one "addon-install-blocked" I am a +1 for the vacuous catch all